src/main/java/systems/crigges/jmpq3/JMpqEditor.java

```diff
@@ -655,28 +696,54 @@ public void insertByteArray(String name, byte[] input) throws NonWritableChannel
         filenameToData.put(name, data);
     }
 
+    /**
+     * Inserts the specified byte array into the mpq once you close the editor.
+     *
+     * @param name  of the file inside the mpq
+     * @param input the input byte array
+     * @throws NonWritableChannelException the non writable channel exception
```

Comment:
For @throws java doc definitions you are supposed to write, why/when this function throws the exception.
The description `the non writable channel exception` is completely pointless - then u might as well omit it completely.

Reply:
You've fixed it